app/src/main/java/org/mozilla/tv/firefox/fxa/FxaRepo.kt

```diff
@@ -44,17 +45,16 @@ class FxaRepo(
 ) {
 
     enum class AccountState {
-        // TODO: Are these states accurate?
         // TODO: Later, may need "failed to login": https://github.com/mozilla-mobile/android-components/issues/3712
-        AUTHENTICATED,
-        AUTHENTICATED_NO_PROFILE, // todo: necessary? profile seems to be fetched async so yes?
-        NEEDS_REAUTHENTICATION,
-        NOT_AUTHENTICATED
-        // TODO: do we need initial state before state is fetched from disk?
+        AUTHENTICATED_WITH_PROFILE, // After the profile is fetched async
+        AUTHENTICATED_NO_PROFILE, // Before the profile is fetched async
+        NEEDS_REAUTHENTICATION, // When an auth error occurs
+        NOT_AUTHENTICATED, // When logged out
+        INITIAL // Behaves the same as NOT_AUTHENTICATED
     }
 
     // TODO: set state accurately, choose correct subject, set initial state, etc.
-    val accountStateDontUseMeYet = BehaviorSubject.create<AccountState>()
+    val accountState = BehaviorSubject.createDefault(INITIAL)
```

Comment:
Nit: we don't want to publicly expose this `Subject`, that would let the rest of the app push state to it.  Could you update this to follow the pattern of exposing the `.hide` val?  See `SessionRepo.state` for an example